libs/base/ui/form/input-list/src/component.ts

```diff
@@ -4,4 +4,9 @@ export const inputListComponent = componentDef({
   name: 'oryx-input-list',
   impl: () =>
     import('./input-list.component').then((m) => m.InputListComponent),
+  stylesheets: [
+    {
+      rules: () => import('./input-list.styles').then((m) => m.screenStyles),
+    },
+  ],
```

Comment:
after using design tokens for typography, we have only 1 rule left, let's inline it here rather than lazy load the css on small screen.

Reply:
Replaced values by tokens

Reply:
You could consider to use:

```ts
${headingUtil(HeadingTag.Subtitle)}
```